main.py

```diff
@@ -85,31 +95,35 @@ def getLabel(label, type='single'):
 
     # Prints length and save as a csv
     print(label, 'Df length:', len(df.index))
-    df.to_csv('data/{}.csv'.format(label))
+    df.to_csv('{}s/{}.csv'.format(label_type, label))
 
     # Save as json object with mapping address:nameTag
-    addressNameDict = dict([(address, nameTag)
+    if label_type == "account":
+        addressNameDict = dict([(address, nameTag)
                            for address, nameTag in zip(df.Address, df['Name Tag'])])
-    with open('data/{}.json'.format(label), 'w', encoding='utf-8') as f:
+    if label_type == "token":
+        addressNameDict = dict([(address, nameTag)
+                           for address, nameTag in zip(df.Address, df['Token Name'])])  
+    with open('{}s/{}.json'.format(label_type, label), 'w', encoding='utf-8') as f:
         json.dump(addressNameDict, f, ensure_ascii=True)
 
-    if (type == 'single'):
+    if (input_type == 'single'):
         endOrContinue = input(
             'Type "exit" end to end or "label" of interest to continue')
         if (endOrContinue == 'exit'):
             driver.close()
         else:
-            getLabel(endOrContinue)
+            getLabel(endOrContinue, label_type=label_type)
 
 # Combines all JSON into a single file combinedLabels.json
 def combineAllJson():
     combinedJSON = {}
 
     # iterating over all files
-    for files in os.listdir('./data'):
+    for files in os.listdir('./accounts'):
         if files.endswith('json'):
             print(files)  # printing file name of desired extension
-            with open('./data/{}'.format(files)) as f:
+            with open('./accounts/{}'.format(files)) as f:
                 dictData = json.load(f)
                 for address, nameTag in dictData.items():
                     if address not in combinedJSON:
```

Comment:
Any thoughts regarding the need to combine for tokens as well or just accounts being sufficient?